content/rs/release-notes/rs-6-0-20-april-2021.md

```diff
@@ -149,6 +152,14 @@ All known bugs around setting ciphers were fixed.  To learn more, see [Updating
 
 -   Envoy updated and verified with multiple security headers.
 
+-   Starting Redis Software 6.0.20:
```

Comment:
When you say "starting" do you mean this is the behavior from 6.0.20 forward, or are you referring to how the customer starts the software?

Reply:
the behavior from 6.0.20 forward